pandas/_libs/tslibs/nattype.pyx

```diff
@@ -19,6 +19,8 @@ from pandas._libs.tslibs.util cimport (
     get_nat, is_integer_object, is_float_object, is_datetime64_object,
     is_timedelta64_object)
 
+import pandas.compat as compat
```

Comment:
we _really_ dont want this import here.  how about `from cpython.version cimport PY_MINOR_VERSION`

Reply:
Fixed.

And just for I know for the next time, why we really don't want this import?

Reply:
we want to keep the dependency structure simple, and _libs, particularly tslibs is at the base of that structure, so avoids imports from elsewhere in pandas